ResearchKit/Common/CEVRKTheme.h

```diff
@@ -35,3 +35,21 @@ typedef NS_ENUM(NSInteger, CEVRKThemeType) {
 - (void)updateTextForContinueButton:(nonnull ORKContinueButton *)continueButton;
 
 @end
+
+
+@protocol CEVRKThemedUIElement <NSObject>
+
+/**
+ Returns a theme for UI styling.
+ 
+ Any UIElement that can be in the responder chain can conform to this protocol
+ and provide a theme for UI customization. Useful for UI elements that are
+ ResearchKit objects or subclasses thereof that may be used outside of standard
+ ResearchKit view hieararchy.
+ 
+ @return Theme for UI styling.
+ */
+
+- (nonnull CEVRKTheme *)theme;
```

Comment:
`theme` is a pretty generic selector name. I'm a little concerned about a naming collision; what would you think of renaming this to something like `cev_theme` or `rkStudioTheme` or `researchKitTheme`?